modules/placekey/redshift/doc/H3_ASPLACEKEY.md

```diff
@@ -0,0 +1,22 @@
+### H3_ASPLACEKEY
+
+{{% bannerNote type="code" %}}
+placekey.H3_ASPLACEKEY(h3index)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns the placekey equivalent to the given H3 index.
+
+* `h3index`: `STRING` H3 identifier.
+
+**Return type**
+
+`STRING`
+
+**Example**
+
+```sql
+SELECT rscarto.placekey.H3_ASPLACEKEY('847b59dffffffff');
```

Comment:
`SELECT placekey.H3_ASPLACEKEY('847b59dffffffff');`
We don't use the database for calling the functions.